src/ScottPlot/Renderable/Legend.cs

```diff
@@ -159,22 +176,43 @@ public Bitmap GetBitmap(bool lowQuality = false, double scale = 1.0)
                         if ((item.markerShape != MarkerShape.none) && (item.markerSize > 0))
                             MarkerTools.DrawMarker(gfx, markerPoint, item.markerShape, MarkerWidth, item.color);
                     }
+                    if (!item.plottableVisible)
+                    {
+                        // prepare values for drawing a rectangle
+                        PointF hideRectOrigin = new PointF(lineX1, locationY + verticalOffset);
+                        SizeF hideRectSize = new SizeF(width, maxLabelHeight);
+                        RectangleF hideRect = new RectangleF(hideRectOrigin, hideRectSize);
+                        gfx.FillRectangle(legendItemHideBrush, hideRect);
+                    }
                 }
             }
         }
 
-        private LegendItem[] LegendItems;
-        public void UpdateLegendItems(IPlottable[] renderables)
+        public void UpdateLegendItems(IPlottable[] renderables, bool includeall = false)
```

Comment:
The `includeall` argument is a little confusing, especially if seen outside the context of this PR

Consider using a more descriptive name like `includeInvisibleAndEmptyLabelPlottables` and/or adding XML docs above this method to clarify what this argument does